### DISCUSSION TITLE
osu!lazer cursor freezing when tablet is set on mouse mode

### DISCUSSION THREAD

**Discussion by 143catstreet** [ppy/osu#26767] [Q&A]
### Type

Game behaviour

### Bug description

My cursor freezes every second while im moving around my cursor using my tablet, my mouse works just fine. I use my tablet on mouse mode and with raw input. I cant play in any other way since this is how i learnt to play on tablet. pls fix

### Screenshots or videos

https://streamable.com/gbntqy

### Version

2024.114.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14076184/compressed-logs.zip)

**Comment by bdach** [comment]
Usual spiel:

- try turning off tablet handler in-game off
- try turning off high precision mouse
- try changing or uninstalling any special tablet drivers you have
